### PR TITLE
Feat/cdp 88

### PR DESCRIPTION
**Pull request by lcnascimento**

### CODE REVIEW

File: lib/http/middleware/brand/brand.go [hunk 1]
```diff
@@ -8,36 +8,43 @@ import (
...
-	CodeTypeMissingLoggerDependency errors.CodeType = "MISSING_LOGGER_DEPENDENCY"
 )
 
-// AccountAuthenticator structure responsible for handling request authentication
+// BrandFiller structure responsible for injecting the brand into HTTP request context.
 type BrandFiller struct {
 	logger logger
 }
 
 // BrandFiller creates a new instance of the Brand structure
```
Comment (victoriavilasb) [inline lib/http/middleware/brand/brand.go:23]:
```suggestion
// NewBrandFiller creates a new instance of the Brand structure
```

File: lib/jwks/jwks.go [hunk 2]
```diff
@@ -57,7 +79,7 @@ func (c *Client) RenewCerts(ctx context.Context) error {
 }
 
 // Certs return a list of valid certs
-func (c Client) Certs() map[string]string {
+func (c *Client) Certs() map[string]string {
```
Comment (victoriavilasb) [inline lib/jwks/jwks.go:82]: Why you changed to a pointer?

File: lib/trace/trace.go [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+	otel.SetTracerProvider(tp)
+	otel.SetTextMapPropagator(propagation.NewCompositeTextMapPropagator(
+		propagation.TraceContext{},
+		propagation.Baggage{},
+	))
+
+	return tp.Tracer(params.ApplicationName), tp.Shutdown, nil
+}
+
+// NewTracer creates a new Tracer.
```
Comment (victoriavilasb) [inline lib/trace/trace.go:61]:
```suggestion
// MustNewTracer creates a new Tracer.
```

**Review by victoriavilasb — CHANGES_REQUESTED** [review]

File: lib/http/middleware/brand/brand.go [hunk 4]
```diff
@@ -8,36 +8,43 @@ import (
 )
 
 // ContextKeyBrandID is the key used to retrieve and save brand into the context
-const ContextKeyBrandID string = "Brand"
+const ContextKeyBrandID string = "brand"
```
Comment (straubartur) [inline lib/http/middleware/brand/brand.go:11]: i think we should go fot the "brand_id" to follow the convency

**Review by straubartur — CHANGES_REQUESTED** [review]

File: lib/jwks/jwks.go (hunk 2 above)
Comment (lcnascimento) [reply lib/jwks/jwks.go:82]: Our new linter found an issue: `RenewCerts passes lock by value: github.com/ditointernet/go-dito/lib/jwks.Client contains sync.Mutex`

**Pull request merged**